next/src/components/ui/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,31 @@
+import Button from '../button/Button';
+import styles from './checkbox.module.scss';
+import { type Props } from './checkbox.types';
+
+export default function Checkbox({ disabled, text, name, register, errors, error = 'to pole jest wymagane' }: Props) {
+  return (
+    <label className={styles.checkbox}>
+      <div className={styles.checkboxWrapper}>
+        <input
+          type='checkbox'
+          {...register}
+        />
+        {text ? (
+          <p dangerouslySetInnerHTML={{ __html: text }} />
+        ) : (
+          <p>
+            <strong>Akceptuję</strong>{' '}
+            <Button
+              href='/pl/polityka-prywatnosci'
+              svg={true}
+              disabled={disabled}
+            >
+              Politykę Prywatności
+            </Button>
+          </p>
+        )}
+      </div>
+      {error && <>{errors[name] && <span className={styles.error}>{error}</span>}</>}
```

Comment:
I would rather have an <Error /> element to handle e.g. animations globally in the future